app/uk/gov/hmrc/individualsemploymentsapi/controller/EmploymentsController.scala

```diff
@@ -37,8 +37,8 @@ abstract class EmploymentsController(employmentsService: EmploymentsService) ext
   def root(matchId: UUID) = Action.async { implicit request =>
     requiresPrivilegedAuthentication {
       employmentsService.resolve(matchId) map { _ =>
-        val payeLink = HalLink("paye", s"/individuals/employments/paye/match?matchId=$matchId{&fromDate,toDate}", title = Option("View individual's employments"))
-        val selfLink = HalLink("self", s"/individuals/employments/match?matchId=$matchId")
+        val payeLink = HalLink("paye", s"/individuals/employments/paye?matchId=$matchId{&fromDate,toDate}", title = Option("View individual's employments"))
+        val selfLink = HalLink("self", s"/individuals/employments?matchId=$matchId")
```

Comment:
needs to be /individuals/employments/?matchId=$matchId here due to wso2 requiring / at then end of api